src/fragments/Graph.react.js

```diff
@@ -352,6 +353,14 @@ class PlotlyGraph extends Component {
         if (this.props.extendData) {
             this.mergeTraces(this.props, 'extendData', 'extendTraces');
         }
+
+        if (
+            (this.props.prependData?.length ?? 0) +
+                (this.props.extendData?.length ?? 0) >
+            0
```

Comment:
Just curious, is there a reason you prefer this to
`if (this.props.prependData?.length || this.props.extendData?.length)`
?

Reply:
None whatsoever, will update.

Reply:
https://github.com/plotly/dash-core-components/pull/865/commits/a156dccb05f916968d0c2170d9a7a86a9b8acb8a